src/Avalonia.Controls/Primitives/RangeBase.cs

```diff
@@ -75,8 +75,11 @@ public double Minimum
 
             set
             {
-                ValidateDouble(value, "Minimum");
-
+                if (!ValidateDouble(value))
+                {
+                    value = _minimum;
```

Comment:
shouldnt it just return at this point and not make any change?

Reply:
i wonder if we could do the same with the Maximum field too?